i18n/designs.mjs

```diff
@@ -23,7 +23,7 @@ import { i18n as examples } from '@freesewing/examples'
 import { i18n as florence } from '@freesewing/florence'
 import { i18n as florent } from '@freesewing/florent'
 import { i18n as gozer } from '@freesewing/gozer'
-import { i18n as hi } from '@freesewing/hi'
+import { i18n as bonny, i18n as hi } from '@freesewing/hi'
```

Comment:
You are importing from hi here? That is probably a mistake?

Please add an import for bonny (these imports are also kept in alphabetical order)

Reply:
Oops.